### PR TITLE
Updated local_training_ios

### PR DESCRIPTION
**Pull request by haripriya9647**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Relevant Issue
Issue #Tag the issue here

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [ ] My changes are covered by tests

### CODE REVIEW

File: docs/source/tutorials/Tutorial-Part-7-local_training_ios.rst [hunk 1]
```diff
@@ -1,2 +1,122 @@
 Local Training on iOS
-=====================
\ No newline at end of file
+=====================
+
+On-device training
```
Comment (ramesht007) [inline docs/source/tutorials/Tutorial-Part-7-local_training_ios.rst:4]: can you add a ```<br>``` here?

File: docs/source/tutorials/Tutorial-Part-7-local_training_ios.rst [hunk 2]
```diff
@@ -1,2 +1,122 @@
...
+Creating new job
+~~~~~~~~~~~~~~~~
+To create a training job locally, you need to supply the model name and version.
+
+.. code:: SwiftSyft
+  self.syftJob = syftClient.newJob(modelName: "mnist", version: "1.0.0")
+
+Training Hooks
+~~~~~~~~~~~~~~
+Here is a function called onReady( ) that's called when SwiftSyft has downloaded plans, parameters from PyGrid and is ready to train the model on your data.The function onReady( )
```
Comment (ramesht007) [inline docs/source/tutorials/Tutorial-Part-7-local_training_ios.rst:28]: can you write this in bullet points?

**Review by ramesht007 — CHANGES_REQUESTED** [review]
hmm, I cannot see the code in the rst file.
is there some problem?

**Review by divyanshugit — CHANGES_REQUESTED** [review]
Please complete these changes

**Pull request merged**